website/client/assets/scss/modal.scss

```diff
@@ -0,0 +1,32 @@
+@mixin centeredModal() {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  header, footer {
+    border: 0;
+  }
+}
+
+.modal-dialog {
+  .title {
+    height: 24px;
+    margin-top: 24px;
+    font-family: 'Roboto Condensed';
+    font-size: 20px;
+    font-weight: bold;
+    line-height: 1.2;
+    text-align: center;
+    color: #4e4a57;
+  }
+
+  .text {
+    height: 60px;
+    font-family: Roboto;
```

Comment:
this is not necessary, also all the height values for text css from Zeplin are not necessary, they're already going to be okay with line-heigth+font-size